__tests__/cake.test.ts

```diff
@@ -1,14 +1,17 @@
-import * as path from 'path';
 import { exec } from '@actions/exec';
 import { which } from '@actions/io';
+import * as path from 'path';
+
 import * as cake from '../src/cake';
-import { ToolsDirectory } from '../src/toolsDirectory';
-import { CakeToolSettings } from '../src/cakeToolSettings';
 import { CakeArgument, CakeSwitch } from '../src/cakeParameter';
+import { CakeToolSettings } from '../src/cakeToolSettings';
+import { ToolsDirectory } from '../src/toolsDirectory';
```

Comment:
**nitpick:** the order of the `import` statements is deliberate. Please, revert this change.

Reply:
Reordered imports to previous commit